src/oatpp/core/data/stream/Stream.cpp

```diff
@@ -408,7 +409,7 @@ v_io_size ConsistentOutputStream::writeAsString(v_float32 value){
 
 v_io_size ConsistentOutputStream::writeAsString(v_float64 value){
   v_char8 a[100];
-  auto size = utils::conversion::float64ToCharSequence(value, &a[0], 100);
+  auto size = parser::json::Utils::float64ToJson(value, &a[0], 100);
```

Comment:
Since `ConsistentOutputStream` is a general-purpose class - it should
use `auto size = utils::conversion::float64ToCharSequence(value, &a[0], 100);` as before